s3backgrounddelete/s3backgrounddelete/cortx_s3_config.py

```diff
@@ -74,7 +74,8 @@ def _load_and_fetch_config(self):
                 " it doesn't have read access")
             sys.exit()
         with open(self._conf_file, 'r') as file_config:
```

Comment:
do not 'open' the file - kick off - confstore class

Reply:
check example code of confstore

Reply:
done